quill/quill.d.ts

```diff
@@ -1,3 +1,8 @@
+// Type definitions for quill.js
+// Project: https://github.com/quilljs/quill/tree/0.20.1
+// Definitions by: Mike Morearty <https://github.com/mmorearty>
```

Comment:
All I did was to move it from our private codebase over to DefinitelyTyped. Please see my private email to you. Thanks!